### DISCUSSION TITLE
nodejs The installation failed

### DISCUSSION THREAD

**Discussion by IggyMartin** [nodejs/help#4711] [Q&A]
### Version

18.16.0

### Platform

Macbook Pro

### What steps will reproduce the bug?

As soon as the installation starts ('Configuring the installation') 

### How often does it reproduce? Is there a required condition?

Every time I try installing Nodejs

### What do you see instead?

A warning message saying "The installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance"

**Comment by bnoordhuis** [comment]
Converted to a discussion because it's unlikely to be a bug in node's installer. It's probably something on your end, like not having the appropriate permissions to install software.
